pretyping/pretyping.ml

```diff
@@ -1198,8 +1198,7 @@ let all_no_fail_flags = default_inference_flags false
 
 let ise_pretype_gen_ctx flags env sigma lvar kind c =
   let evd, c = ise_pretype_gen flags env sigma lvar kind c in
-  let evd, f = Evarutil.nf_evars_and_universes evd in
```

Comment:
Removing this universe minimisation is potentially dangerous but if things work without it I'm happy.
@mattam82 WDYT?